src/neuroglancer/widget/segment_set_widget.ts

```diff
@@ -224,10 +262,21 @@ export class SegmentSetWidget extends RefCounted {
         itemButton.addEventListener('mouseenter', function(this: HTMLButtonElement) {
           temp.tryParseString(this.textContent!);
           widget.segmentSelectionState.set(temp);
+          widget.segmentSelectionState.setRaw(temp);
+          const segmentButton =
+              <HTMLElement>document.querySelector(`[data-segID="${temp.toString()}"]`);
+          segmentButton.className += ' selectedSeg';
         });
         itemButton.addEventListener('mouseleave', function(this: HTMLButtonElement) {
           temp.tryParseString(this.textContent!);
           widget.segmentSelectionState.set(null);
+          widget.segmentSelectionState.setRaw(null);
+          const segmentButton =
+              <HTMLElement>document.querySelector(`[data-segID="${temp.toString()}"]`);
+          let classes = Array.from(segmentButton.classList);
```

Comment:
See above, there is already a function you can use for this whole block: `segmentButton.className.remove('selectedSeg');`.